lib/cog_api/resources/role.ex

```diff
@@ -4,5 +4,6 @@ defmodule CogApi.Resources.Role do
   defstruct [
     :id,
     :name,
+    :permissions,
```

Comment:
This works when I have tested by hand. I would like to rerecord using exvcr and see how that works.